source/sass/components/chapter-page.scss

```diff
@@ -18,3 +18,28 @@
 }
 
 // .chapter-article-link {}
+
```

Comment:
Just kinda stuck these here for now...but we could just rename this file to publication-page and stick styles here. Also does it count as a component? I'd expect component files to be smaller elements of this page and not styles for larger elements/the page itself. We can sort this out later though.